libs/partners/cohere/langchain_cohere/embeddings.py

```diff
@@ -26,9 +26,9 @@ class CohereEmbeddings(BaseModel, Embeddings):
             )
     """
 
-    client: Any  #: :meta private:
+    client: Optional[cohere.Client]  #: :meta private:
```

Comment:
```suggestion
    client: cohere.Client = Field(default=None)  #: :meta private:
```

Reply:
This is how we typically do it. Is there a case where you actually don't want `self.client` to be defined?

Reply:
Looks like in `validate_environment` it is always initialized anyways, so probably better to just type it as such!

Reply:
that makes sense, will do that pattern, thanks!